src/Pickles/Pickles.MSBuild/Pickles.cs

```diff
@@ -51,6 +51,8 @@ public class Pickles : Task
 
         public string EnableComments { get; set; }
 
+        public string IgnoreTag { get; set; }
```

Comment:
In terms of the intent of the API, I think that "ExcludeTags" is a better name for the property. Right now we support only one tag but I still want to call the property with the plural "Tags" to ensure forward compatibility.

Please rename the property to ExcludeTags (and adapt the rest of the code).